### PR TITLE
ft-313 Add mochatto.com url to README and github link to header

### LINKED ISSUE #313: [Feature]: Add mochatto.com url to README and github link to footer
### What feature?

Add the mochatto url to README
Add the github icon/link to this page as a footer

### PR DESCRIPTION
**Pull request by siberowl**
## Which Issue was this PR made for?

- Closes #313 

## What is within the scope with this PR
-Adding website link to readme
-Adding github link to website

image

## Other sidenotes that reviewer should be aware of
Refactored to remove some atomize stuff.

### CODE REVIEW

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-332.dev.mochatto.com

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-332.dev.mochatto.com

File: client/src/components/Logo/index.tsx [hunk 1]
```diff
@@ -3,14 +3,12 @@ import React from "react";
 import { Div } from "atomize";
```
Comment (Njima1572) [inline client/src/components/Logo/index.tsx:3]: Unused Div (Yes we are getting out of it

File: client/src/assets/imgs/GitHub-Mark/.DS_Store:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (Njima1572) [inline client/src/assets/imgs/GitHub-Mark/.DS_Store:Zone.Identifier:1]: What are those files for??

File: client/src/templates/BaseTemplate/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Div, Row, Col } from "atomize";
+import { Logo } from "@/components";
+import "./style.scss";
+import githubLogo from "../../assets/imgs/GitHub-Mark/PNG/GitHub-Mark-120px-plus.png";
```
Comment (Njima1572) [inline client/src/templates/BaseTemplate/index.tsx:6]: If we are only using one image, then just keep one and remove the others. (Might be nice to keep both light/dark version though)

File: client/src/templates/BaseTemplate/style.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 8px;
+
+  .logo, .github-logo {
+    cursor: pointer;
+  }
+  img {
+    height: 25px;
```
Comment (Njima1572) [inline client/src/templates/BaseTemplate/style.scss:10]: `.header img` is overriding `.logo-img`, not sure if this was your intention, if it was then you could probably remove `.logo-img`

image

**Review by Njima1572 — CHANGES_REQUESTED** [review]

File: client/src/assets/imgs/GitHub-Mark/.DS_Store:Zone.Identifier (hunk 2 above)
Comment (siberowl) [reply client/src/assets/imgs/GitHub-Mark/.DS_Store:Zone.Identifier:1]: Looks like it was autogenerated somehow. They're not needed so I'll delete them.

File: client/src/templates/BaseTemplate/index.tsx (hunk 3 above)
Comment (siberowl) [reply client/src/templates/BaseTemplate/index.tsx:6]: I agree yeah

File: client/src/templates/BaseTemplate/style.scss (hunk 4 above)
Comment (siberowl) [reply client/src/templates/BaseTemplate/style.scss:10]: Good catch

Comment (siberowl) [reply client/src/templates/BaseTemplate/style.scss:10]: Nvm it's still being used in the room template.

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-332.dev.mochatto.com

**Review by Njima1572 — APPROVED** [review]

**Comment by Njima1572** [comment]
LGTM

**Pull request merged**